### PR TITLE
EFA-262 Allow nullability in fields to avoid crash

### PR DESCRIPTION
**Pull request by ig-garcia**

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/message/MessageBuilder.java [hunk 1]
```diff
@@ -640,7 +640,7 @@ public MessageBuilder setIdentity(foundation.pEp.jniadapter.Identity from, Addre
         com.fsck.k9.Identity k9Identity = new com.fsck.k9.Identity();
         k9Identity.setSignatureUse(false);
         k9Identity.setSignature("");
-        k9Identity.setReplyTo(PlanckUtils.getReplyTo(replyTo));
+        k9Identity.setReplyTo(replyTo == null ? null : PlanckUtils.getReplyTo(replyTo));
```
Comment (Stereodark) [inline k9mail/src/main/java/com/fsck/k9/message/MessageBuilder.java:643]: and what happens when it is null? I mean if you set it to null, isn't there any other side effect? What would be the use case?

**Review by Stereodark — COMMENTED** [review]
yep make sense

**Review by pzplanck — APPROVED** [review]

File: k9mail/src/main/java/com/fsck/k9/message/MessageBuilder.java (hunk 1 above)
Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/message/MessageBuilder.java:643]: Well setting a field to null is the same as if it was not initialized.
The uses are:
- storing in SharedPreferences -> Storing null for a String is ok.
- Displaying this String on screen -> null String is ok for a view text.

So I see no issues.

Comment (Stereodark) [reply k9mail/src/main/java/com/fsck/k9/message/MessageBuilder.java:643]: my main concern was having a K9Identity with a replyTo field == null. If we use that field somewhere assuming it is not null, we will get null pointers, the question was more "do we use k9Identity anywhere assuming replyTo is not null"? If not this is ok.

**Pull request merged**